src/datagen/generated/mekanism/assets/minecraft/atlases/blocks.json

```diff
@@ -50,15 +50,15 @@
     },
     {
       "type": "minecraft:single",
-      "resource": "mekanism:infuse_type/bio"
```

Comment:
uhh why did these change order? were they not being sorted before?

Reply:
They weren't exactly being sorted by us, and when I switched to iterating the registry by entryset instead of value, it changed. This is in part due to me having messed up the switch. I then noticed it when doing the code review of the PR myself, and fixed that, but I decided to keep in the sorting logic I added just so that it is a bit more consistent overall.